pkg/koordlet/resourceexecutor/executor.go

```diff
@@ -110,6 +121,11 @@ func (e *ResourceUpdateExecutorImpl) UpdateBatch(cacheable bool, updaters ...Res
 func (e *ResourceUpdateExecutorImpl) LeveledUpdateBatch(cacheable bool, updaters [][]ResourceUpdater) {
```

Comment:
remove cacheable since it is never supported?

Reply:
done